notion/collection.py

```diff
@@ -465,11 +507,14 @@ def _convert_python_to_notion(self, val, prop, identifier="<unknown>"):
         if prop["type"] in ["title", "text"]:
             if not val:
                 val = ""
-            if not isinstance(val, str):
+            if not isinstance(val, (str,list)):
```

Comment:
Nit: space after commas (but I'll do a pass with `black` after which will clean things up).

Reply:
Also, I totally forgot that you can pass a tuple of types to `isinstance` -- nice!

Reply:
Can you clarify the case in which we'd be passing in a list here, though? Is this for when you'd want to use the native/internal notion format externally?

Reply:
Same situation I didn'ŧ realize that I made pull request from master. Maybe I will do another pull request for this feature. But the main idea is, that I want to pass format that is not representable by markdown - e,g, text and backgroud colors, user mentions, dates etc. `[text, ['h', color]]`, `[' @' + client.get_user(user_id).full_name, [['b']]]`